### ISSUE TITLE
After new users join a room, their messages don't reach IRC users

### ISSUE THREAD

**Issue by tadzik** [matrix-org/matrix-ircd#23]

**Comment by madduck** [comment]
Related to #8 maybe?

**Comment by tadzik** [comment]
Yep, it gets fixed by proper JOIN handling. I'm not sure if that's merged yet? I have my local patch for it, but I don't think it's quite ready for prime time: https://github.com/tadzik/matrix-ircd/tree/gh-23-membership-events

**Comment by madduck** [comment]
I have not encountered this problem, but I'll  keep an eye out. Might be 933b2442dccab019a9ba24c61cac4046f1667a07 or  ff52e76e813275e79edad6207fe4f522d4a6624a?

Now where is the per-github-issue-matrix-room I could join instead of having to deal with the web UI? ;)

**Comment by tadzik** [comment]
Ah, yes. Hm. Maybe my patch is merged then? %)

**Comment by madduck** [comment]
you're best to judge this yourself. ;)

**Comment by jplatte** [comment]
~~@madduck From one of the other issues it seems you've kept using matrix-ircd for quite a while after your last comment here. Has this problem ever appeared again?~~

Sorry, it wasn't you who opened this issue. Seems like the PR in question is #28, which hasn't been merged yet.
